packages/bpk-component-info-banner/src/BpkInfoBannerInner.tsx

```diff
@@ -107,7 +107,9 @@ const ToggleButton = (props: ToggleButtonProps) => {
       aria-expanded={props.expanded}
       title={props.label}
     >
-      <ExpandIcon className={classNames} />
+      <div className={classNames}>
+        {props.expanded ? <ExpandIcon/> : <ContractIcon/>}
```

Comment:
With the above suggestion
```suggestion
        {props.expanded ? <ExpandIcon/> : <CollapseIcon/>}
```